samples/canopen/canopen_rpdo/TestCanNode.cpp

```diff
@@ -0,0 +1,30 @@
+#include "TestCanNode.hpp"
```

Comment:
Replace this with a `<>` include

Reply:
Why? I am including a local file in the same directory not a standard or added library like EVT-core.

Reply:
Yeah, I just assumed this was a mistake, but it does make sense to use quotes here. This is a unique case where the .hpp and .cpp are in the same directory, so it should be fine.